ds-prep-python-sets/main.py

```diff
@@ -0,0 +1,25 @@
+trailmix = {"m&ms", "peanuts", "raisins"}
+print(trailmix)
+print("cashews" in trailmix)
+print("peanuts" in trailmix)
+trailmix.add("pretzels")
+trailmix.update(["peanuts", "banana chips", "bits of jerky"])
```

Comment:
While this does technically follow the directions, this whole exercise is about practicing with `set`s, so I'd suggest trying this with a `set` instead of a `list`.

---

![image](https://user-images.githubusercontent.com/16143771/149263181-f81d21c6-a0b7-4832-8f77-43da2b7a3004.png)